orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -146,6 +146,16 @@ class SqlExecutionRepository(
     storeStage(stage)
   }
 
+  override fun updateStageOthers(stage: StageExecution) {
+    storeStage(stage)
```

Comment:
This is just boiling down to saving the entire stage ... why bother introducing "others" (which is likely going to be confusing) instead of just storing in the stage relevant stage context.